RELEASE_NOTES.md

```diff
@@ -7,19 +7,21 @@ A new header is inserted each time a *tag* is created.
 
 ## v1.9.21
 
+- engine: Add new `Renderer::renderStandaloneView()` API.
```

Comment:
Please add this under the `## Next release (main branch)` header. It will go out with v1.9.22.

Reply:
wow good catch!